packages/platform/example/lib/main.dart

```diff
@@ -0,0 +1,149 @@
+                FormatDetails(
+                  title: 'Path Separator:',
+                  value: platform.pathSeparator,
+                ),
+                FormatDetails(
+                  title: 'Local Hostname:',
+                  value: platform.localHostname,
+                ),
+                FormatDetails(
+                  title: 'Environment:',
+                  value: platform.environment.toString(),
+                ),
+                FormatDetails(
+                  title: 'Executable:',
+                  value: platform.executable,
+                ),
+                FormatDetails(
+                  title: 'Resolved Executable:',
+                  value: platform.resolvedExecutable,
+                ),
+                FormatDetails(
+                  title: 'Script:',
+                  value: platform.script.toString(),
+                ),
+                FormatDetails(
+                  title: 'Executable Arguments:',
+                  value: platform.executableArguments.toString(),
+                ),
+                FormatDetails(
+                  title: 'Package Config:',
+                  value: platform.packageConfig.toString(),
+                ),
+                FormatDetails(
+                  title: 'Version:',
+                  value: platform.version,
+                ),
+                FormatDetails(
+                  title: 'Stdin Supports ANSI:',
+                  value: platform.stdinSupportsAnsi.toString(),
+                ),
+                FormatDetails(
+                  title: 'Stdout Supports ANSI:',
+                  value: platform.stdoutSupportsAnsi.toString(),
+                ),
+                FormatDetails(
+                  title: 'Locale Name:',
+                  value: platform.localeName,
+                ),
+                FormatDetails(
+                  title: 'isAndroid:',
+                  value: platform.isAndroid.toString(),
+                ),
+                FormatDetails(
+                  title: 'isFuchsia:',
+                  value: platform.isFuchsia.toString(),
+                ),
+                FormatDetails(
+                  title: 'isIOS:',
+                  value: platform.isIOS.toString(),
+                ),
+                FormatDetails(
+                  title: 'isLinux:',
+                  value: platform.isLinux.toString(),
+                ),
+                FormatDetails(
+                  title: 'isMacOS:',
+                  value: platform.isMacOS.toString(),
+                ),
+                FormatDetails(
+                  title: 'isWindows:',
+                  value: platform.isWindows.toString(),
+                ),
+              ],
+            ),
+          ),
+        ),
+      ),
+    );
+  }
+}
+
+/// A widget to format the details.
+class FormatDetails extends StatelessWidget {
+  /// Constructs a [FormatDetails]
+  /// [title] is the title of the field.
+  /// [value] is the value of the field.
```

Comment:
These two lines can be removed. The property comments are enough in this context; an example app doesn't need the same level of IDE-targeted doc comments that the framework does.